tests/migration/sle12_online_migration/yast2_migration.pm

```diff
@@ -17,10 +17,17 @@ use utils;
 use power_action_utils 'power_action';
 use version_utils 'is_desktop_installed';
 
+sub yast2_migration_s390x_gnome {
+    return check_var('MIGRATION_METHOD', 'yast') && check_var('ARCH', 's390x') && check_var('DESKTOP', 'gnome');
```

Comment:
Actually I think it is not an arch-specific thing but depending on just the remote control over VNC, right?

@SergioAtSUSE this seems to be related what you had planned with a variable for "remote backend".

I recommend to set a corresponding variable whenever we use VNC remote control instead of a local connection and then check for this variable.

E.g. in *main.pm*:

```perl
set_var('REMOTE_CONNECTION', 'vnc') if get_var('BACKEND', '') =~ /s390x/svirt/ipmi/;
```

and in *tests/migration/sle12_online_migration/yast2_migration.pm*:

```perl
sub yast2_migration_gnome_remote {
    return check_var('MIGRATION_METHOD', 'yast') check_var('DESKTOP', 'gnome') && get_var('REMOTE_CONNECTION');
}
```

Reply:
I didn't plan a variable, but a function.
Adding yet another variable is artificial, since "remote backend" and "remote installation" are defined by other characteristics, it should be ask by those characteristics instead of implementing another high level logic on openQA (machines, test suites)

Reply:
fixed